plugin/trino-hive/src/main/java/io/trino/plugin/hive/PropertiesSystemTableProvider.java

```diff
@@ -60,6 +63,9 @@
         if (table.isEmpty()) {
             throw new TableNotFoundException(tableName);
         }
+        if (isDeltaLakeTable(table.get())) {
+            throw new TrinoException(HIVE_UNSUPPORTED_FORMAT, "Cannot query Delta Lake table");
```

Comment:
nit: would be good to mention table name in error message

Reply:
I modified the exception message to its original format to be in sync with the other places where the exception is being thrown.

Reply:
I agree with @phd3  we should provide table name in the message.

Reply:
Good point. I'll make a separate commit then to update everywhere the exception message.

Reply:
agreed, the `io.trino.plugin.hive.HiveMetadata#getTableHandle` can (and should) be updated in a separate commit.